pkg/options/unity.go

```diff
@@ -0,0 +1,24 @@
+package options
+
+import "github.com/bugsnag/bugsnag-cli/pkg/utils"
+
+// UnityLineMapping is used to specify options for uploading a Unity il2cpp mapping file.
+type UnityLineMapping struct {
+	NoUploadIl2cppMappingFile bool       `help:"Do not upload the il2cpp mapping file"`
+	UploadIl2cppMappingFile   utils.Path `help:"The path to the il2cpp mapping file to upload"`
```

Comment:
Is there anyway to stop the option being called `--no-upload-il-2-cpp-mapping-file` and put the `il2cpp` together?

If not, I'd rather call this `NoUploadLineNumberMappingsFile` / `LineNumberMappingsFile`